apps/prairielearn/src/tests/sprocs/users_select_or_insert.test.js

```diff
@@ -67,7 +67,7 @@ describe('sproc users_select_or_insert tests', () => {
       name: 'J.R. User',
     };
 
-    const result = await usersSelectOrInsert(user, 'Shibboleth');
+    const result = await usersSelectOrInsert(user, 'SAML');
```

Comment:
It doesn't currently affect the outcome of the test, but the `SAML` authn provider will in practice always be used in combination with an explicit institution ID. Consider omitting this second argument or changing it to something like `Google`.

Reply:
I removed it in b65159b. It's tricky because 'Shibboleth' is the de facto default authn provider and that one got converted in the mass search/replace. I agree for that test the authn provider isn't necessary.